auth_otp/services/auth_service.py

```
@@ -0,0 +1,437 @@
+                )
+                
+                # غیرفعال کردن verification مربوطه
+                OTPVerification.objects.filter(
+                    user=user,
+                    refresh_token=refresh_token,
+                    is_active=True
+                ).update(is_active=False)
+                
+                logger.info(f"User {user.username} logged out from single device")
+            
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)